app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ExposureNotificationRepository.kt

```diff
@@ -220,28 +218,11 @@ class ExposureNotificationRepositoryImpl(
     }
 
     override suspend fun isAppRegisteredForExposureNotificationsCurrentState(): Boolean {
-        return suspendCancellableCoroutine { cancellableContinuation ->
-            exposureNotificationClient.isEnabled
-                .addOnSuccessListener {
-                    cancellableContinuation.resume(it)
-                }
-                .addOnFailureListener {
-                    Timber.e(SilentError(it))
-                    cancellableContinuation.resume(false)
-                }
-        }
+        return exposureNotificationClient.isEnabled.await()
```

Comment:
semantic change!! for exception

Reply:
@dusanjencik

Reply:
Fixed. Getter also refresh the subject to have the latest state up to date.